### PR TITLE
Add support of Orange Pi3 board

### PR DESCRIPTION
**Pull request by Vitalii55**
Signed-off-by: Vitalii Tykhenko <vitaliyt75@gmail.com>

# Description

Add support of new Orange Pi3 board.
It is interesting to know how easy it is to run a project on various mini SBC

## Type of change

Please delete options that are not relevant.

- Documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration. Below is just an example and you can delete it when you contribute.

```
Installed docker and Go to Orange Pi3 board
Performed "git clone https://github.com/lf-edge/edge-home-orchestration-go.git"
Performed build and run project on Orange Pi3 board
Checked log by command "docker logs -f edge-orchestration "
```

**Test Configuration**:
* Firmware version: Ubuntu 20.04.2, kernel orangepi3 5.4.65-sunxi64
* Hardware: CPU (H6 Quad-core 64-bit 1.8GHZ ARM Cortex™-A53)
* Toolchain: (e.g., Docker and Go 16.2 arm64)
* Edge Orchestration Release: (Coconut)

# Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have performed a self-review of support of the new Orange Pi3 board

### CODE REVIEW

**Review by MoonkiHong — COMMENTED** [review]
Welcome Onboard! @Vitalii55

File: docs/platforms/orange_pi3/orange_pi3.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+#### 2. Start Orange Pi 3
+
+Insert the SD card into the Orange Pi 3 and turn on the power. Make configuration settings for the Orange the first time you turn it on.
+
+---
+
+## Quick start
+This section provides how to download and run pre-built Docker image without building the project.
+
+> TBD
```
Comment (MoonkiHong) [inline docs/platforms/orange_pi3/orange_pi3.md:22]: Better to indicate how initiate the starting point here to build up the HomeEdge on OrangePi. Any idea on this?

**Review by MoonkiHong — COMMENTED** [review]
Please take a look at the comment. We are open to your opinion.

File: docs/platforms/orange_pi3/orange_pi3.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+```shell
+$ ./build.sh container arm64
+```
+or for protected mode:
+```shell
+$ ./build.sh container secure arm64
+```
+the build result will be `edge-orchestration.tar` archive that can be found `bin/edge-orchestration.tar`
+
+Next, need to copy `edge-orchestration.tar` archive to the Paspberry Pi 3 board, install the docker container (see [here](../x86_64_linux/x86_64_linux.md#Build-Prerequisites) only docker part) and load the image using the command:
```
Comment (smarthome-hanium) [inline docs/platforms/orange_pi3/orange_pi3.md:47]: Please fix a typo error.
```suggestion
Next, need to copy `edge-orchestration.tar` archive to the Orange Pi3 board, install the docker container (see [here](../x86_64_linux/x86_64_linux.md#Build-Prerequisites) only docker part) and load the image using the command:
```

**Review by smarthome-hanium — COMMENTED** [review]
Thanks for your contribution!
Please take a look at my comment.

File: docs/platforms/orange_pi3/orange_pi3.md (hunk 2 above)
Comment (smarthome-hanium) [inline docs/platforms/orange_pi3/orange_pi3.md:47]:
```suggestion
Next, need to copy `edge-orchestration.tar` archive to the Raspberry Pi 3 board, install the docker container (see [here](../x86_64_linux/x86_64_linux.md#Build-Prerequisites) only docker part) and load the image using the command:
```

Comment (Vitalii55) [reply docs/platforms/orange_pi3/orange_pi3.md:47]: Fixed, Thanks for you comment

**Review by MoonkiHong — APPROVED** [review]
LGTM.

**Comment by tdrozdovsky** [comment]
I checked this request, everything works well. LGTM

**Comment by t25kim** [comment]
I was waiting to fix https://github.com/lf-edge/edge-home-orchestration-go/pull/293#discussion_r610350627
Someone will fix it later. 😊

**Pull request merged**